r/R/array.R

```diff
@@ -291,3 +299,11 @@ is.Array <- function(x, type = NULL) {
   }
   is_it
 }
+
+#' @export
+sort.Array <- function(x, decreasing = FALSE, na.last = TRUE, ...) {
```

Comment:
Because the method for `Scalar` is defined differently
https://github.com/apache/arrow/pull/9745/commits/003b6f7036601f8444cc305bf394b8806b0d9062#diff-631d7e73ccb129db2ba09fe78b6a26e7164476e6142e926ba0fb4753599b7026R73

Reply:
That's fine, S3 inheritance will handle that right?

Reply:
Oh right, ok, I'll check that and change it if it works 👍

Reply:
Done in 09356cf237996ec3ca497eecc941603b9af98fdd